pkg/moq/moq.go

```diff
@@ -291,3 +318,27 @@ var templateFuncs = template.FuncMap{
 		return strings.ToUpper(s[0:1]) + s[1:]
 	},
 }
+
+func gopaths() []string {
+	return strings.Split(os.Getenv("GOPATH"), string(filepath.ListSeparator))
+}
+
+// StripGopath teks the directory to a package and remove the gopath to get the
+// canonical package name.
+func StripGopath(p string) string {
+	for _, gopath := range gopaths() {
+		p = strings.TrimPrefix(p, path.Join(gopath, "src")+"/")
+	}
+	return p
+}
+
+
+// StripVendorPath splits the path on the first vendor dir and returns
+// everything after that. I guess this is the vendor dir for the project...
+func StripVendorPath(p string) string {
+	parts := strings.Split(p, "/vendor/")
+	if len(parts) == 1 {
+		return p
+	}
+	return strings.TrimLeft(path.Join(parts[1:]...), "/")
+}
```

Comment:
> I guess this is the vendor dir for the project...

I would remove comments like these since it lowers the collective confidence in the code. Same with the commit message. If you believe this PR fixes the problem then remove the "maybe".